_data/sites.json

```diff
@@ -5641,10 +5641,10 @@
 
     {
         "name": "MyAnimeList",
-        "url": "https://myanimelist.net/about.php?go=contact",
+        "url": "https://myanimelist.net/account_deletion",
         "difficulty": "hard",
```

Comment:
Does that mean that the difficulty is not hard anymore, most of the times?

Reply:
Yes, I believe that most users will not need to use the contact form to request deletion of their accounts. Do you think it would be more relevant to set the difficulty of this entry to `medium` instead of `hard?`